config/jsdoc/api/plugins/doc.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Define an @doc tag
+ */
+var conf = env.conf.stability;
+var defaultLevels = ["deprecated","experimental","unstable","stable","frozen","locked"];
```

Comment:
in which case, maybe we should just remove `stable` from typedefs (and other non-api types)